sktime/forecasting/naive_variance.py

```diff
@@ -0,0 +1,192 @@
+# -*- coding: utf-8 -*-
+
+"""Implements naive variance functionality to tune forecasters."""
+
+__author__ = ["IlyasMoutawwakil"]
+
+
+import numpy as np
+import pandas as pd
+from scipy.stats import norm
+from sklearn.base import clone
+
+from sktime.forecasting.base import BaseForecaster
+from sktime.utils.validation.forecasting import check_alpha, check_fh
+
+
+class NaiveVariance(BaseForecaster):
+    """
+    Compute the prediction variance using the 'simple strategy'.
```

Comment:
put the math here, or it´s totally unclear what this implements.

Reply:
I put a description but I'm not sure if what you mean is to put latex formulas